### PR TITLE
[WIP] fix crash

### PR DESCRIPTION
**Pull request by huiping192**
crash detail:
`validateRenderPassDescriptor:950: failed assertion No rendertargets set in RenderPassDescriptor.'

### CODE REVIEW

File: Kitsunebi/Classes/AnimationView.swift [hunk 1]
```diff
@@ -86,8 +87,9 @@ open class PlayerView: UIView {
     gpuLayer.drawsAsynchronously = true
     gpuLayer.contentsGravity = .resizeAspectFill
     gpuLayer.pixelFormat = .bgra8Unorm
-    gpuLayer.framebufferOnly = false
+    gpuLayer.framebufferOnly = true
     gpuLayer.presentsWithTransaction = false
+    (gpuLayer as? CAMetalLayer)?.device = MTLCreateSystemDefaultDevice()
```
Comment (noppefoxwolf) [inline Kitsunebi/Classes/AnimationView.swift]: See my comment above. ⬆️

File: Kitsunebi/Classes/AnimationView.swift [hunk 2]
```diff
@@ -64,8 +64,9 @@ open class PlayerView: UIView {
...
```
Comment (noppefoxwolf) [inline Kitsunebi/Classes/AnimationView.swift]: Really need to create device?
```suggestion
    (gpuLayer as? CAMetalLayer)?.device = device
```

**Comment by huiping192** [comment]
I have not idea how to fix the crash,so I just close the pull request. Sorry,man

**Comment by noppefoxwolf** [comment]
Thank you @huiping192 !

I fixed this issue.
https://github.com/noppefoxwolf/Kitsunebi/commit/39947c6a4ad627a83bf96df5d4031ec36dc0a2fd

**Pull request closed without merging**